lib/index.js

```diff
@@ -47,6 +47,13 @@ function getTimestamp() {
     return year + month + day + "_" +  hour + minute + second;
 }
 
+function _clean_body(body) {
+    // External links with target="_blank" won't open in default
+    // browser
+    cheerio('a[target="_blank"]', body).removeAttr('target');
+    return body;
+}
+
```

Comment:
@manuq I think there's already a method to do this in libingester iirc and we keep adding this to individual ingesters - can we move this functionality fully to libingester?